nipype/info.py

```diff
@@ -141,7 +141,8 @@ def get_nipype_gitversion():
     'configparser',
     'pytest>=%s' % PYTEST_MIN_VERSION,
     'mock',
-    'pydotplus'
+    'pydotplus',
+    'packaging',
```

Comment:
Building the docker image (I never figured out how to do virtualenvs) from scratch. Is that an equivalent check?